### PR TITLE
Changed from app into pages

### PR DESCRIPTION
**Pull request by staszekz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "preinstall": "npx only-allow pnpm"
```
Comment (EMalanczak) [inline package.json:11]: whats does it do?

File: pages/index.tsx [hunk 2]
```diff
@@ -1,6 +1,7 @@
 'use client';
```
Comment (EMalanczak) [inline pages/index.tsx]: I think we don't want to render whole pages on the client side, do we?

File: src/components/card/card.tsx [hunk 3]
```diff
@@ -1,11 +1,12 @@
 'use client';
+
 import { Card, Group, Image, Text, Badge } from '@mantine/core';
 
 import { CardProps } from '@types';
 
-import { ShoppingListTable } from './shopping-list-table';
+import { ShoppingListTable } from '../shopping-list-table/shopping-list-table';
```
Comment (EMalanczak) [inline src/components/card/card.tsx]: I personally don't like such long imports - what do you think about exporting from `index` files?

**Review by EMalanczak — COMMENTED** [review]
Project structure has unnessesary folders eg. `/components/navigation/navigation.tsx`. It's not worth introducing another folder nesting until there's specific reason for that eg. component becomes big enough so we want to split it into subfiles for greater readability

File: package.json (hunk 1 above)
Comment (staszekz) [reply package.json:11]: prevents from installing dependencies by something else than pnpm.

File: pages/index.tsx (hunk 2 above)
Comment (staszekz) [reply pages/index.tsx]: yes

File: src/components/card/card.tsx (hunk 3 above)
Comment (staszekz) [reply src/components/card/card.tsx]: fixed

**Review by EMalanczak — APPROVED** [review]

**Review by EmilMalanczak — APPROVED** [review]

**Pull request merged**